src/WCSimSteppingAction.cc

```diff
@@ -100,7 +100,7 @@ G4int WCSimSteppingAction::G4ThreeVectorToWireTime(G4ThreeVector *pos3d,
 } 
 
 
-void WCSimSteppingAction::Distortion(G4double x,G4double y)
+void WCSimSteppingAction::Distortion(G4double /*x*/,G4double /*y*/)
```

Comment:
Did you mean to comment out the x and y here? It doesn't currently affect anything since this entire routine is commented out, but this should be left as is unless there is some reason to change it for this PR.